frontend/src/common/Header/Login.jsx

```diff
@@ -29,6 +29,8 @@ const Login = ({ handleLogin, displayProfile }) => {
         console.log(authResult.credential.idToken); // eslint-disable-line no-console
         console.log(redirectUrl); // eslint-disable-line no-console
         document.cookie = `threaderAuthToken=${authResult.user.xa}`;
+        const storedPostTypes = JSON.parse(localStorage.getItem('firebaseui::rememberedAccounts'));
+        console.log(storedPostTypes);
```

Comment:
// eslint-disable-next-line no-console
if you have to have the console logs, other wise just get rid of this?